blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -99,12 +139,145 @@ const LargePromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <LargePromoPresentation content={content} {...customFields} />;
+	// const { id } = useFusionContext();
+	const { editableContent, searchableField } = useEditableContent();
+	const { registerSuccessEvent } = useComponentContext();
+	const {
+		dateLocalization: { language, timeZone, dateTimeFormat } = {
+			language: "en",
+			timeZone: "GMT",
+			dateFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+		},
+	} = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(getProperties(arcSite).locale || "en");
+	const bylineNodes = formatAuthors(content?.credits?.by, phrases.t("global.and-text"));
+
+	// show the override url over the content image if it's present
+	// get the image from content if no override
+	const targetImage = imageOverrideURL || getImageFromANS(content);
+
+	// Start Overline data
+	const {
+		display: labelDisplay,
+		url: labelUrl,
+		text: labelText,
+	} = (content?.label && content?.label?.basic) || {};
+	const shouldUseLabel = !!labelDisplay;
+
+	const { _id: sectionUrl, name: sectionText } =
+		(content?.websites &&
+			content?.websites[arcSite] &&
+			content?.websites[arcSite].website_section) ||
+		{};
+
+	// Default to websites object data
+	let [text, url] = [sectionText, sectionUrl];
+
+	if (content?.owner?.sponsored) {
+		text = content?.label?.basic?.text || phrases.t("overline.sponsored-content");
+		url = null;
+	} else if (shouldUseLabel) {
+		[text, url] = [labelText, labelUrl];
+	}
+	// End Overline data
+
+	const displayDate = localizeDateTime(
+		new Date(content?.display_date),
+		dateTimeFormat,
+		language,
+		timeZone
+	);
+
+	const editableDescription = content?.description
+		? editableContent(content, "description.basic")
+		: {};
+
+	return (
+		<HeadingSection>
+			<Grid as="article" className={BLOCK_CLASS_NAME}>
+				{showImage ? (
+					<MediaItem
+						className={`${BLOCK_CLASS_NAME}__media`}
+						{...searchableField("imageURL")}
+						suppressContentEditableWarning
+					>
+						<Conditional
+							component={Link}
+							condition={content?.websites?.[arcSite]?.website_url}
+							href={formatURL(content?.websites?.[arcSite]?.website_url)}
+							onClick={registerSuccessEvent}
+							assistiveHidden
+						>
+							{playVideoInPlace ? (
+								<Video
+									aspectRatio={aspectRatio}
+									className={`${BLOCK_CLASS_NAME}__video`}
```

Comment:
```suggestion
```
Can style this via the `component: ( video: () )` styling if needed.

Reply:
@nschubach : Thank you for pointing these out! I was looking for exactly this. I think it makes sense it use components and minimize hooks when appropriate. There are differences in the large and extra large versions of the blocks and so the Sass hooks vs. `components` usage will probably differ. Just wanted to make sure we were staying consistent in overall approach.
@malavikakoppula : Do you have a good feeling on the approach for when to use Sass selectors for a block and when to use `component()` properties?